README.md

```diff
@@ -68,48 +69,34 @@ const BlockService = require('ipfs-block-service')
 const BlockService = require('ipfs-block-service')
 const Block = require('ipfs-block')
 const IPFSRepo = require('ipfs-repo')  // storage repo
-const Store = require('interface-pull-blob-store')  // in-memory store
 
 // setup a repo
-var repo = new IPFSRepo('example', { stores: Store })
+const repo = new IPFSRepo('example')
 
 // create a block
-const block = new Block('hello world')
-console.log(block.data)
-console.log(block.key())
-
-// create a service
-const bs = new BlockService(repo)
-
-// add the block, then retrieve it
-bs.put({
-  block: block,
-  cid: cid,
-}, function (err) {
-  bs.get(cid, function (err, b) {
-    console.log(block.data.toString() === b.data.toString())
+const data = new Buffer('hello world')
+multihashing(data, 'sha2-256', (err, hash) => {
+  if (err) throw err
+
+  const block = new Block(data, new CID(hash))
+
+  // create a service
+  const bs = new BlockService(repo)
+
+  // add the block, then retrieve it
+  bs.put(block, (err) => {
+    if (err) throw err
+    bs.get(block.cid, (err, b) => {
```

Comment:
and use the cid from the var, just to make it obvious that you don't need to get the cid from the block.